action.yml

```diff
@@ -0,0 +1,9 @@
+name: "Black"
+description: "The uncompromising Python code formatter."
+author: "Łukasz Langa and contributors to Black"
+branding:
+  color: "black"
```

Comment:
Could be worth testing in case the docs are outdated, but https://help.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#branding doesn't list black:

> The background color of the badge. Can be one of: white, yellow, blue, green, orange, red, purple, or gray-dark.

Reply:
Quickly tested this, it works (I also immediately removed that test too).

![2020-06-01T08:51:09,257888262+02:00](https://user-images.githubusercontent.com/730811/83383302-5d2f5a00-a3d4-11ea-84f0-b3bd7f845a07.png)